MAINTENANCE.md

```diff
@@ -0,0 +1,72 @@
+# Maintainer Guide 
+
+This file outlines the current maintainer(s) of this open source project and expectations. It also includes credits to past maintainers and the project creator. 
+
+## Project Name 
+
+When reference externally, please use this for the short name: 
+
+- Production Docker deployment for Mattermost 
+
+Please use this long name: 
+
+- Production Docker deployment for Mattermost by the Mattermost open source project
+
+## Maintainer(s)
+
+The following people help to maintain this open source project: 
+
+| Current Maintainer(s)  | Start Date    | 
+|:-----------------------|:--------------|
+| Pan Luo - @xcompass    | Nov 30 2015   |
+
+In case something happens where a maintainer is unable to complete their responsibilies or find a new maintainer, the following sponsoring organization can help find a new maintainer: 
```

Comment:
If you want keep "their", then you may want to change "where a maintainer is unable" to "where maintainers are unable" to be consistent with "their"